### ISSUE TITLE
Fix pkgdown configuration for reference

### ISSUE THREAD

**Issue by maelle** [ropensci/osfr#144]
Some topics are missing from the configuration file.

```r
 Error in check_missing_topics(rows, pkg) : 
  All topics must be included in reference index
• Missing topics: osf_cp, osfr-package
```

Note that for topics you do not want to include in the index you can create an "internal" section https://pkgdown.r-lib.org/reference/build_reference.html?q=internal#missing-topics

To check all topics are listed, after editing the configuration file you can run `pkgdown::check_pkgdown()`.

**Comment by maelle** [comment]
@aaronwolen in case you missed this (not that urgent but probably easy to fix)

**Comment by aaronwolen** [comment]
Thank you for reporting @maelle! I'll try to get that fixed.

**Comment by maelle** [comment]
Thank you!

**Comment by aaronwolen** [comment]
Fixed in #147
